src/handlers/motions/motionFinalized/helpers.ts

```diff
@@ -49,3 +52,48 @@ export const getStakerReward = async (
     isClaimed: false,
   };
 };
+
+export const getParsedActionFromMotion = async (
+  motionAction: string,
+  colonyAddress: string,
+): Promise<TransactionDescription | undefined> => {
+  const colonyClient = await networkClient.getColonyClient(colonyAddress);
+
+  // We are only parsing this action in order to know if it's a add/edit domain motion. Therefore, we shouldn't need to try with any other client.
+  try {
+    return colonyClient.interface.parseTransaction({
+      data: motionAction,
+    });    
+  } catch {
+    verbose(`Unable to parse ${motionAction} using colony client`);
+    return undefined;
+  }
+};
+
+export const linkPendingDomainMetadataWithDomain = async (action: string, colonyAddress: string, finalizedMotion: MotionQuery) => {
```

Comment:
Getting some linting errors on this function (no return type) and prettier also showing me squiggles. Could you install eslint / prettier and run so we keep everything consistent?

Reply:
This will happen eventually #31